src/ecrc-frontend/src/components/base/textInput/TextInput.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./TextInput.css";
+
+export const TextInput = ({
+  textInput: { label, id, textInputStyle, value, isRequired }
+}) => {
+  let redStar = "";
+  if (isRequired === true) {
```

Comment:
this can just be changed to:
```suggestion
  if (isRequired) {
```

Reply:
since it is a truthy value

Reply:
yes.